assets/sass/components/_result-card.scss

```diff
@@ -0,0 +1,18 @@
+.result-card {
+  border: 1px solid govuk-colour('mid-grey');
+  margin-bottom: govuk-spacing(6);
+
+  &__header {
+    border-bottom: 1px solid govuk-colour('mid-grey');
+    background-color: govuk-colour("light-grey");
+    padding: govuk-spacing(4);
+
+    & .govuk-heading-m {
+      margin-bottom: 0;
+    }
```

Comment:
Is this needed?

Could we not add `govuk-!-margin--bottom-0`  to the headings that need margin 0? Theres a really good write up https://design-system.service.gov.uk/get-started/extending-and-modifying-components/